test-suite/rpath_hack.py

```diff
@@ -0,0 +1,12 @@
+import glob
+import subprocess
+import sys
+
+pattern = "${CMAKE_CURRENT_BINARY_DIR}/PyCFFIlib_cffi*.so"
+for so in glob.glob(pattern):
+    print("rpath hack %s" % (so))
+    rpath_hack = subprocess.run(
+        ["${CMAKE_INSTALL_NAME_TOOL}", "-change", "@rpath/$<TARGET_FILE_NAME:mylib>", "${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_FILE_NAME:mylib>", so])
+    if rpath_hack.returncode != 0:
+        sys.exit(rpath_hack.returncode)
```

Comment:
I remember @a4z offering a solution that does not require this hack? Am I correct with that? Is there still plans to get rid of this?

Reply:
I had forgotten about his PR 😬 ... fixed now, thanks!